infra/scripts/language/agents/order_refund_plugin.py

```diff
@@ -0,0 +1,13 @@
+from semantic_kernel.functions import kernel_function

```

Comment:
How come we have dupes of this here and in `src/backend/agents`? Can we just import or copy the files over for prod?